### DISCUSSION TITLE
Older Versions (Lua5.1)

### DISCUSSION THREAD

**Discussion by LedLoaf** [microsoft/vcpkg#18811] [Q&A]
Hello,

So that last few weeks I've been trying to understand the concept of vcpkg (_amazing!_), which lead me to discover similar managers. I like this one the best, but I just needed to use ubuntu to get a linux program to run and I noticed they can choose specific versions of the libraries. 

With that said, I'm having trouble finding the answer to my problem, I'm wondering if I can get earlier versions of libraries? I did a search and I'm coming across some conflicting information from a few years ago. I saw something about using 'version' and I was hoping someone could tell me if this is possible with todays version; no pun intended? Because from what I gathered this wasn't possible 2-3 years ago, but I have a hunch it is now.

Thanks, have a great rest of weekend and be safe.

**Comment by autoantwort** [comment]
See [this](https://github.com/microsoft/vcpkg/discussions/17161#discussioncomment-586714) post
